launch_testing/test/check_env.py

```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    import os
+    assert(os.environ.get('testenv2', 'Not Set') == 'Not Set')
```

Comment:
Makes sense. Changed in https://github.com/ros2/launch/pull/38/commits/ede0634c469bb3a12e3f89a6b2faec93a2be4dfd